### DISCUSSION TITLE
tkinterdnd2

### DISCUSSION THREAD

**Discussion by avalon60** [Akascape/Easy-Tier-List#3] [General]
Hi @Akascape. Hope that things are all good with you!

Just wondering if you use Intel based Mac? The reason I ask, is that I noticed that you are using tkinterdnd2 module, and as I understand it tkinterdnd2-universal is required for ARM based Macs. I was looking at tkinterdnd2-universal for this reason, but unfortunately it won't install to my Linux env. Is this something that you have experimented with?

**Comment by Akascape** [comment]
@avalon60 No I haven't tried tkinterdnd2 on mac/linux. I tested it only on windows.

**Reply by avalon60** [reply]
Seems I had a corrupted venv, stopping me from installing modules.
